core/block/import/txt/converter.go

```diff
@@ -162,3 +164,9 @@ func (t *TXT) getSnapshot(blocks []*model.Block, p string) (*converter.Snapshot,
 	}
 	return snapshot, snapshot.Id
 }
+
+func (t *TXT) shouldReturnError(req *pb.RpcObjectImportRequest, cErr *converter.ConvertError, path []string) bool {
+	return (!cErr.IsEmpty() && req.Mode == pb.RpcObjectImportRequest_ALL_OR_NOTHING) ||
+		(cErr.IsNoObjectToImportError(len(path))) ||
```

Comment:
I think outer braces are redundant here. It is a style thing, but this complex logical expression will look a little bit more readable without them.
Moreover `&&` is computed before `||` , so return value could be simplified to:
```
return !cErr.IsEmpty() && req.Mode == pb.RpcObjectImportRequest_ALL_OR_NOTHING ||
		cErr.IsNoObjectToImportError(len(path)) || 
		errors.Is(cErr.GetResultError(pb.RpcObjectImportRequest_Txt), converter.ErrCancel)
```

Reply:
Removed this code and move it to `ShouldAbortImport` function in `ConveterError`